app/main.py

```diff
@@ -1,3 +1,16 @@
-def cache(func):
-    # Write your code here
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    cache_result = {}
+
+    def wrapper(*args) -> Callable:
+        cache_key = (args)
```

Comment:
Why you need this line?

Reply:
you are right, I don't need this one. Fixed